js/app.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener(`DOMContentLoaded`, () =>{
+
+	const form = document.querySelector(`#registrar`);
+	const formInput = form.querySelector(`input`);
+	const invitedUl = document.querySelector(`#invitedList`);
+	const hideNotResponded = document.querySelector(`.main input`);
+
+	function newElement(elementParent, element) {
+		const name = document.createElement(element);
+		elementParent.appendChild(name)
+		return name
+	}
+
+	function text(elementParent, element, text) {
+		const name = newElement(elementParent, element);
+		name.textContent = text;
+		return name;
+	}
+
+	function type(elementParent, element, type) {
+		const name = newElement(elementParent, element);
+		name.type = type;
+		return name;
+	}
+
+	function submitForm() {
+		const newLi = newElement(invitedUl, 'li');
+		const newSpan = text(newLi, `span`, formInput.value);
+		const newInput = type(newLi, `input`, 'text');
+		newInput.value = formInput.value;
+		newInput.classList.add(`invisible`);
+		const label = text(newLi, `label`, `Confirmed`);
+		label.setAttribute(`for`, `confirmed`);
+		const checkbox = type(label, 'input', `checkbox`);
+		checkbox.setAttribute(`name`, `confirmed`);
+		const edit = text(newLi, 'button', 'Edit');
+		const remove = text(newLi, 'button', `Remove`);
+	}
+
+
+	form.addEventListener(`submit`, (e) => {
+		e.preventDefault();
+		submitForm();
+		formInput.value = ``;
+	});
+
+	invitedUl.addEventListener('change', (e) => e.target
+			.parentElement
+			.parentElement
+			.classList.toggle(`responded`));
```

Comment:
Jeśli konicznie nie chcesz dawać tutaj `{}` to trzymaj wszystko w jednej linijce i nie rozbijaj na kolejne